jbpm-designer-client/src/main/java/org/jbpm/designer/client/util/ListBoxValues.java

```diff
@@ -71,11 +69,26 @@ public ListBoxValues(final String customPrompt,
         this.maxDisplayLength = DEFAULT_MAX_DISPLAY_LENGTH;
     }
 
+    public ListBoxValues(final ListBoxValues copy,
+                         final boolean copyCustomValues) {
+        this.customPrompt = copy.customPrompt;
+        this.editPrefix = copy.editPrefix;
+        this.valueTester = copy.valueTester;
+        this.maxDisplayLength = copy.maxDisplayLength;
+        this.addValues(copy.acceptableValuesWithoutCustomValues);
+        if (copyCustomValues) {
+            for (String copyCustomValue : copy.customValues) {
+                this.addCustomValue(copyCustomValue,
+                                    null);
+            }
+        }
+    }
```

Comment:
@hasys yeah i think it is. in designer custom values are displayed as individual properties as well. when they change this copy should be made (old value really does not matter imo so its ok)